.gitignore

```diff
@@ -1,7 +1,10 @@
 *.o
-
+*.valgrind
+*.vscode
 /configure
 /Makefile
 /config.cache
 /BUILD
 /INST
+vscode
+
```

Comment:
This change really belong in a separate PR - it's nothing to do with adding these new operators at all.

Please don't reformat whitespace in existing files when modifying them (here you removed an existing blank line separately glob patterns and verbatim names, and added an extra blank line at the end.

`*.valgrind` looks wrong to me - did you actually mean to add `.valgrind.log.*`?  The test harness creates those as temporary files, but it should also remove them again (unless interrupted).  We didn't add them to `.gitignore` previously because it seems better to know they've been left behind - otherwise you can end up with a lot of diskspace used by these files as the name includes the process id.  I found I had thousands at one point, though we now remove them on "make clean" which helps avoid them building up indefinitely.